src/envo/envo-edit.owl

```diff
@@ -3535,6 +3553,15 @@ Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510038>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510039>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510040>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510041>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510055>))
```

Comment:
>Masonry cement which comprises a mixture of elastic polymers mixed in a solvent.

Rephrase to "Masonry cement which is composed primarily of elastic polymers mixed in a solvent"

The axioms include 
* `alcohol` as a part, but this is not stated in the definition or comment. If this is not true for all instances of `rubber cement` this should be removed. 
* `rubber particle`  - this does not align to the textual definition. Consider modifying the textual definition to include it or remove it.

Reply:
The definition has been revised for 'rubber cement' to reflect this recommendation. Axioms regarding the inclusion of 'rubber particle' and 'alcohol' have been maintained (as they are accurate for all instances of rubber cement) and can be added to the textual definition if required.